tests/integration/test_cloudformation.py

```diff
@@ -1737,3 +1739,20 @@ def cleanup(self, stack_name, change_set_name=None):
 
     def expected_change_set_status(self):
         return 'CREATE_COMPLETE'
+
+    def test_list_exports_correctly_returns_exports(self):
+        stack_name = 'stack-%s' % short_uid()
+
+        template = load_file(TEST_TEMPLATE_27)
+
+        deploy_cf_stack(stack_name, template_body=template)
+
+        cloudformation = aws_stack.connect_to_service('cloudformation')
+        response = cloudformation.list_exports()
+
+        exports = response['Exports']
+        self.assertEqual(len(exports), 1)
+        self.assertEqual(exports[0]['Name'], 'T27SQSQueue1URL')
```

Comment:
Thanks for this PR @apalumbo ! 

Looks like we still have a minor build error: https://travis-ci.org/github/localstack/localstack/builds/753363011 . Should this assertion be defined as follows instead?
```
self.assertEqual(exports[0]['Name'], 'SQSQueue1-URL')
```

Reply:
@whummer yes, sorry I changed it to avoid race condition as once a test fail it will not do the cleanup ;-). good catch!